tests/rkt-monitor/main.go

```diff
@@ -127,21 +134,25 @@ func runRktMonitor(cmd *cobra.Command, args []string) {
 
 	var rktBinary string
 	if flagRktDir != "" {
-		rktBinary = flagRktDir + "/rkt"
+		rktBinary = filepath.Join(flagRktDir, "rkt")
+		if !fileExist(rktBinary) {
+			fmt.Fprintln(os.Stderr, "rkt binary not found!")
+			os.Exit(1)
+		}
 	} else {
 		rktBinary = "rkt"
 	}
 
 	for i := 0; i < flagRepetitionNumber; i++ {
-		containerStarting = time.Now()
-
 		// build argument list for execCmd
 		argv := []string{"run"}
 
 		if flagStage1Path != "" {
 			argv = append(argv, fmt.Sprintf("--stage1-path=%v", flagStage1Path))
 		}
 
+		argv = append(argv, "--debug")
```

Comment:
tiny nit: you could put this on the line that argv gets created on, and you'd save a line